File: Domain.AggregatesModel/EstudianteCursoAgreggate/EstudianteCurso.cs
namespace Domain.AggregatesModel
{
    public class EstudianteCurso
    {
        public int Id { get; set; }
        public int IdEstudiante { get; set; }    
        public int IdCurso { get; set; }       
        public double? NotaFinal { get; set; }

        public EstudianteCurso() { }

        public EstudianteCurso(int id, int idEstudiante, int idCurso, double? notaFinal)
        {
            Id = id;
            IdEstudiante = idEstudiante;
            IdCurso = idCurso;
            NotaFinal = notaFinal;
        }
    }
}

File: colegioAppWeb/src/app/models/estudianteCurso.ts
export class EstudianteCurso{
    id: number;
    idEstudiante : number;
    idCurso : number;
    notaFinal?: number;
   
    constructor(id:number, idEstudiante:number, idCurso:number, notaFinal:number) { 
        this.id = id; 
        this.idEstudiante = idEstudiante; 
        this.idCurso = idCurso; 
        this.notaFinal = notaFinal; 
     } 
}